modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -367,6 +367,183 @@ public void testGetProps() throws Exception {
 		}
 	}
 
+	@Test
+	public void testGetPropsWithAssetCategoriesAndChildAssetCategoriesSortedByKey()
```

Comment:
We have a really big tests, could we simplify this test into smaller ones?.

What happens when you have 1 Audience with 2 journal, and 1 Stage with 1 journal...

Reply:
well, i could test the order of Audience (x-axis vocab) in one test and another test for stage (y-axis vocab). But, i wanted to test the big picture in this test

Reply:
Usually it is not a good idea to have a big test, because when it fails, who know what the cause is, we should decompose all the cases in smaller ones.

Reply:
A test should fails only because a cause.

Reply:
Done, I've created 2 separate class, thanks a lot!